crates/goose-server/src/state.rs

```diff
@@ -42,4 +44,19 @@ impl AppState {
             .clone()
             .ok_or_else(|| anyhow::anyhow!("Scheduler not initialized"))
     }
+
+    /// Reset the app state
+    /// after we make goosed multi-agent, we shouldn't need this anymore
+    pub async fn reset(&self) {
+        let mut agent = self.agent.lock().await;
+        *agent = Some(Arc::new(Agent::new()));
+
+        {
+            let mut scheduler = self.scheduler.lock().await;
+            *scheduler = None;
+        }
```

Comment:
the PR mentions recipes but this looks like it only resets the scheduler?

Reply:
we don't keep track of the recipes on the server side yet, so the reset of that still happens on the client sadly